plugins/modules/docker_container.py

```diff
@@ -2641,6 +2670,26 @@ def _get_expected_binds(self, image):
         self.log(result, pretty_print=True)
         return result
 
+    def _get_expected_labels(self, image):
+        if self.parameters.image_label_mismatch == 'ignore':
+            expected_labels = self._get_image_labels(image)
+        else:
+            expected_labels = {}
+        if self.parameters.labels:
+            expected_labels.update(self.parameters.labels)
```

Comment:
Then it's also no longer necessary to check this one:
```suggestion
        expected_labels.update(self.parameters.labels)
```